Prox/Prox/RemoteConfigKeys.swift

```diff
@@ -70,19 +71,62 @@ class RemoteConfigKeys {
 }
 
 extension RemoteConfigKeys {
-    open static func getDouble(forKey key: String) -> Double {
-        return FIRRemoteConfig.remoteConfig()[key].numberValue!.doubleValue
+    open static func getTimeInterval(forKey key: String) -> TimeInterval {
+        return FIRRemoteConfig.remoteConfig()[key].numberValue!.doubleValue * 60
     }
+}
+
+class RemoteConfigProperty<T> {
+    let key: String
+    let defaultValue: T
 
-    open static func getString(forKey key: String) -> String {
-        return FIRRemoteConfig.remoteConfig()[key].stringValue!
+    var value: T {
+        let remoteConfig = FIRRemoteConfig.remoteConfig()
+        let rcv = remoteConfig[key]
+        if let value = convert(rcv) {
+            return value
+        }
+        if let rcv = remoteConfig.defaultValue(forKey: key, namespace: nil),
+            let value = convert(rcv) {
+            return value
+        }
+        return defaultValue
```

Comment:
One comes from swift, one comes from the plist.

I'm not altogether happy with having a plist at all, but current usage for an array of very long strings is done my constructing keys. Rather than a larger scale refactor of that, I commented on this, and continued with supporting the plist.

I've added a print statement to show where the default is coming from.